main/src/main/java/com/sedmelluq/discord/lavaplayer/source/twitch/TwitchStreamAudioSourceManager.java

```diff
@@ -70,7 +70,8 @@ public String getSourceName() {
 
     @Override
     public AudioItem loadItem(AudioPlayerManager manager, AudioReference reference) {
-        String streamName = getChannelIdentifierFromUrl(reference.identifier);
+	    // Using root because the turkish lowercase "i" does not have the little dot above the letter when defaulted
+        String streamName = getChannelIdentifierFromUrl(reference.identifier).toLowerCase(Locale.ROOT);
```

Comment:
Actually on second thoughts, it's probably better to lowercase after the null check otherwise this could throw a `NullPointerException`.

Reply:
Yeah true